components/ProfilePageComponent/form/ProfileSummaryForm.tsx

```diff
@@ -0,0 +1,150 @@
+import { ErrorMessage, Field, Form, FormikProvider, useFormik } from 'formik';
+import React from 'react';
+import Styled from '@emotion/styled';
+import * as Yup from 'yup';
+import { isPhoneNumber } from 'js-string-helper';
+import { getPhoneNumberRegex } from '../../../helper/regex';
+import { css, cx } from '@emotion/css';
+import Button, { ButtonType } from '../../common/Button';
+import { PutData } from '../../../Utils/fetchData';
+import { UseAppSelector } from '../../../store';
+import { getUserState } from '../../../store/user/basicInfo';
+
+const FormContainer = Styled.div`
+    padding: 20px;
+`;
+const FieldGrid = Styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 15px;
+`;
+
+const InputFieldContainer = css`
+    width: 100%;
+    padding: 12px;
+`;
+
+const InputField = css`
+    width: 100%;
+    padding: 12px;
+    border-radius: 8px;
+    border: 1px solid gray;
+    display: block;
+`;
+
+const ErrorMessageClass = css`
+    margin-top: 8px;
+    color: red;
+    font-weight: 300;
+    font-size: 14px;
+`;
+
+const ButtonContainer = Styled.div`
+    display: flex;
+    justify-content: flex-end;
+`;
+
+const Heading = Styled.h4`
+    font-size: 24px;
+    font-weight: 600;
+`;
+
+const validationSchema = Yup.object().shape({
+    profileHeading: Yup.string(),
+    phoneNumber: Yup.string()
+        .matches(getPhoneNumberRegex, "Doesn't look like a phone number")
+        .nullable(),
+    designation: Yup.string(),
+    country: Yup.string().required('Required'),
+    city: Yup.string().required('Required'),
+});
+
+const ProfileSummaryForm: React.FC<{ closeModal: () => void }> = ({ closeModal }) => {
+    const { _id = '' } = UseAppSelector(getUserState);
+
+    const formik = useFormik({
+        initialValues: {
+            profileHeading: '',
```

Comment:
@JRakin how do you manage edit??

Reply:
@sifatul ok got it i will update initValue